### ISSUE TITLE
Microsoft.JSInterop.JSException: Couldn't find the editor with id: entity-view editors.length: 0

### ISSUE THREAD

**Issue by FLAMESpl** [serdarciplak/BlazorMonaco#133]
I cannot set value to code editor, I am getting this exception: `Microsoft.JSInterop.JSException: Couldn't find the editor with id: entity-view editors.length: 0`

I am declaring code editor like this:

```
                <StandaloneCodeEditor @ref="codeEditor"
                                      Id="entity-view"
                                      ConstructionOptions="GetConstructionOptions"/>
```

and setting value like this:

```
        await codeEditor.SetValue(args.Json);
```

**Comment by cwevers** [comment]
Same here

**Comment by leigh-pointer** [comment]
@FLAMESpl try setting the value in AfterRender, then you can be pretty sure the interop is loaded.

**Comment by FLAMESpl** [comment]
It is assigned on button click after I stay on a page for few minutes.

**Comment by burtonrodman** [comment]
I am having same issue.  In my situation, I am using a Blazor WebAssembly project and I have a page that I visit multiple times to edit the details of an item.  If I use ConstructionOptions, the value is set on the first instantiation, but then remains the same in subsequent visits to the page -- even though I see the OnInitializedAsync and OnParametersSetAsync get called and I am getting the correct parameters, loading the data and updating the models -- The editor never gets re-initialized with the new value.

If I remove the Value from the ConstructionOptions and try to set it in OnAfterRenderAsync, I get the error (regardless of firstRender).

**Comment by burtonrodman** [comment]
I was able to resolve the issue by setting Value to a single space in the ContructionOptions, and also handle OnDidInit to set the value via SetValue method after my data was loaded.

This seems like a bug on 2 points...
- Calling SetValue after initialization when ConstructionOptions.Value is empty throws the error
- Visiting a page containing the editor doesn't refresh the value when provided by ConstructionOptions.

**Comment by serdarciplak** [comment]
- I cannot reproduce the first issue in the sample net9.0 web app in the repo. It works no matter what value is provided in the `ConstructionOptions`. So, I'm closing this issue. But please reopen it if you can provide a sample project to reproduce the issue, or you can describe how I can reproduce it.
- The second one about refreshing the value is fixed with #137 in `v3.3.0`
